source/crud/update.txt

```diff
@@ -62,17 +62,17 @@ document that matches a specified filter.
    To update the *first* document in the ``sample_mflix.movies``
    collection where ``title`` equals ``"Tag"``:
 
-   .. code-block::
+   .. code-block:: javascript
 
       use sample_mflix
 
-      db.movies.updateOne( { title: "Tag" },
+      db.movies.updateOne( { title: "Twilight" },
       {
         $set: {
-          plot: "One month every year, five highly competitive friends
-                 hit the ground running for a no-holds-barred game of tag"
-        }
-        { $currentDate: { lastUpdated: true } }
+          plot: "A teenage girl risks everything-including her life-when she
```

Comment:
[nit] I think these should be em dashes instead of en dashes!

```suggestion
          plot: "A teenage girl risks everything—including her life—when she
```